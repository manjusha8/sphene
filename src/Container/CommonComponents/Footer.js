import React, { useState } from "react";
import {
  Wrapper,
  FooterWrapper,
  AboutWrapper,
  Title,
  Text,
  HelpWrapper,
  ListContainer,
  ListItems,
  Link,
  Container,
  LeftWrapper,
  RightWrapper,
  ContactWrapper,
  ContactText,
  Copyrights,
  Sphene,
} from "./FooterStyle";

function Footer() {
  const [helpfulLinks] = useState([
    "About Us",
    "Press Realese",
    "Careers",
    "Services",
    "Projects",
  ]);

  const [helpful] = useState([
    "In The News",
    "Our Blogs",
    "Testimonials",
    "Contact",
  ]);

  return (
    <Wrapper>
      <FooterWrapper>
        <AboutWrapper>
          <Title>ABOUT</Title>
          <Text>
            Morbi convallis bibendum urna ut viverra. Maecenas quis consequat
            libero, a feugiat eros. Nunc ut lacinia tortor morbi ultricies
            laoreet ullamcorper phasellus semper.
          </Text>
        </AboutWrapper>
        <HelpWrapper>
          <Title>HELPFUL LINKS</Title>
          <Container>
            <LeftWrapper>
              <ListContainer>
                {helpfulLinks.map((value, index) => (
                  <ListItems key={index}>
                    <Link href="#">{value}</Link>
                  </ListItems>
                ))}
              </ListContainer>
            </LeftWrapper>
            <RightWrapper>
              <ListContainer>
                {helpful.map((value, index) => (
                  <ListItems key={index}>
                    <Link href="#">{value}</Link>
                  </ListItems>
                ))}
              </ListContainer>
            </RightWrapper>
          </Container>
        </HelpWrapper>
        <ContactWrapper>
          <Title>CONTACT US</Title>
          <ContactText>
            <div>12345 Little Lonsdale St, Melbourne</div>
            <div> Phone: (123) 123-456</div>
            <div> Fax: (123) 123-456 </div>
            <div> E-Mail: office@example.com</div>
          </ContactText>
        </ContactWrapper>
      </FooterWrapper>
      <Copyrights>
        Â© Copyright 2016 by <Sphene href="#"> Sphene </Sphene>. All Rights
        Reserved.
      </Copyrights>
    </Wrapper>
  );
}

export default Footer;
