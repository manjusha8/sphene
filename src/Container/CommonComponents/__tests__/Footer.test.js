import React from "react";

import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import Footer from "../Footer";

configure({ adapter: new Adapter() });

describe("Palette Compnenet", () => {
  it("should test the Footer Snapshot ", () => {
    const wrapper = shallow(<Footer />);

    expect(wrapper).toMatchSnapshot();
  });
});
