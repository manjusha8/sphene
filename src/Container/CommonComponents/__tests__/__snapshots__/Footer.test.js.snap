// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Palette Compnenet should test the Footer Snapshot  1`] = `ShallowWrapper {}`;
